build.sh

```diff
@@ -2,14 +2,13 @@
 
 # For GNU Linux
 go build -ldflags="-s -w" -o ./build/tgpt-linux-amd64
```

Comment:
Is there any particular reason for omitting this line?

Reply:
sorry i forget add to this line :)